### PR TITLE
Remote trigger right after sdk is available

### LINKED ISSUE #499: Update the remote trigger jck tests jobs per platforms
Currently AQA_Test_Pipeline job in Eclipse Temurin is remotely triggered by pipeline job when all platforms jdks are available. And then AQA_Test_Pipeline job will trigger all test jobs with all available jdks. It means all jck jobs can only be triggered when all platforms jdks are available. If some platform builds are slow then all tck jobs have to wait.

To run the jck test jobs ASAP we'd like to trigger the AQA_Test_Pipeline job per platform. So jck test jobs can be triggered as long as jdks are available.

### PR DESCRIPTION
**Pull request by sophia-guo**
Fix #499 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by zdtsw** [comment]
@sophia-guo to have a quick test, you can use ""run tests quick 19" than the full tests on all jdk versions with "run tests"

**Comment by sophia-guo** [comment]
The PR tests failures is not related with this PR.

The PR is testing by https://ci.adoptopenjdk.net/view/work-in-progress/job/sophia_buildPipeline/45/

**Review by smlambert — APPROVED** [review]
Some minor grammar/spelling nits, otherwise LGTM

**Comment by zdtsw** [comment]
run tests quick 19

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -1742,6 +1768,24 @@ class Build {
                 if (enableTests) {
                     try {
                         runSmokeTests()
+                        // Remote trigger eclispe temurin jck tests
+                        context.echo " let's try remote trigger"
+                        if (buildConfig.VARIANT == 'temurin' && isRelease) {
+                            context.echo " it's temurin and release"
```
[the remark this answers is not in the document]
Comment (sophia-guo) [reply pipelines/build/common/openjdk_build_pipeline.groovy:1774]: That's for testing, forgot to remove. Will update.

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -1742,6 +1768,22 @@ class Build {
                 if (enableTests) {
                     try {
                         runSmokeTests()
+                        // Remote trigger Eclispe temurin jck tests
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:1771]:
```suggestion
                        // Remote trigger Eclipse Temurin JCK tests
```

**Review by karianna — APPROVED** [review]

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 2 above)
Comment (sophia-guo) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: Updated.

**Pull request merged**